app/models/image.rb

```diff
@@ -1,8 +1,7 @@
 class Image < ActiveRecord::Base
   has_many :attachments, as: :asset
   has_many :sources, through: :attachments
-  validates :file_base, presence: :true
-  validates :mime_type, presence: :true
+  validates :file_name, presence: :true
   validates :height, numericality: { only_integer: true }
   validates :width, numericality: { only_integer: true }
```

Comment:
OK, I'll fix that.  I think I have it setting an empty string, but it's still a good thing to do this.